Demo/src/main/java/com/paypal/android/usecase/CreatePayPalPaymentTokenUseCase.kt

```diff
@@ -0,0 +1,37 @@
+package com.paypal.android.usecase
+
+import com.google.gson.JsonObject
+import com.google.gson.JsonParser
+import com.paypal.android.api.model.PayPalPaymentToken
+import com.paypal.android.api.model.PayPalSetupToken
+import com.paypal.android.api.services.SDKSampleServerAPI
+import org.json.JSONObject
+import javax.inject.Inject
+
+class CreatePayPalPaymentTokenUseCase @Inject constructor(
+    private val sdkSampleServerAPI: SDKSampleServerAPI
+) {
+
+    suspend operator fun invoke(setupToken: PayPalSetupToken): PayPalPaymentToken {
+        // language=JSON
+        val request = """
```

Comment:
Maybe a utils class for all of these JSON objects? Might be helpful for merchants to see the difference side by side (for Card vs PayPal)

Reply:
Yeah I agree it is very loosely-typed right now, mostly because we don't ship the Demo app we made a tradeoff for flexibility early on. But now that the app is getting large it makes sense to refactor a bit.

Reply:
I tried using `kotlinx.serialization` here and I ultimately decided to backtrack. There are some limitations that I'd like to have a larger discussion about with everyone, and see if we can land on a set of JSON serialization best practices.